components/ciphertool/src/main/java/org/wso2/ciphertool/utils/Utils.java

```diff
@@ -414,6 +495,51 @@ public static Map<String, String> getSecreteFromConfiguration(String configFileP
         return context;
     }
 
+    /**
+     * Read from toml file and return keystore data.
+     *
+     * @param configFilePath    file path to deployment toml
+     * @param keystoreName      name of the keystore
+     * @return      Map of keystore data
+     */
+    public static Map<String, String> getKeystoreFromConfiguration(String configFilePath, String keystoreName) {
+        Map<String, String> context = new LinkedHashMap<>();
+        try {
+            TomlParseResult result = Toml.parse(Paths.get(configFilePath));
+            if (result.hasErrors()) {
+                throw new CipherToolException("Error while parsing TOML config file");
+            }
+            TomlTable table = result.getTable(keystoreName);
+            if (table != null) {
+                table.dottedKeySet().forEach(key -> context.put(key, table.getString(key)));
+            }
+
+        } catch (IOException e) {
+            System.out.println("Error parsing file " + configFilePath + e.toString());
+            return context;
+        }
+        return context;
+    }
+
+    /**
+     * Read from default.json.

```

Comment:
Improve this method description

Reply:
Addressed by https://github.com/wso2/cipher-tool/pull/76